next.config.js

```diff
@@ -2,8 +2,6 @@
 const withMDX = require("@next/mdx")();
 
 const nextConfig = {
-  output: "export",
```

Comment:
yeah, I know, we will need to change deployment since the whole routing feature needs the use of node.js. At least that's the explanation I found when looking into dynamic routes. 

<img width="478" alt="image" src="https://github.com/DeXter-on-Radix/website/assets/34665379/e4a0f702-7d81-43a9-b700-cd40d69c7c2f">

Reply:
yes, another way we could try this is using query params as suggested by @dcts 
I just did it this way because it was more readable and since this is my first time working with next.js I could implement this using dynamic routes. Only later did I discover that this only works when removing the export function :)